file.json

```diff
@@ -0,0 +1 @@
+{name = "Jaap Singh"}
```

Comment:
Congrats on the first ever PR! (at least what GitHub told me)

Valid JSON file is tricky!
- [ ] The key part (left hand side) has to be quoted, and the key, value delimiter is colon `:`. It would look something like this:
```
{
    "thisKey" : "thatValue"
}
```

- [ ] Also move this file under `data` folder.